FMS: losing limbs (#79277)
CleverRaven/Cataclysm-DDA: doc/FREQUENTLY_MADE_SUGGESTIONS.md

As of 2024, there have been some very exciting changes to how we handle characters, allowing them to have non-humanoid limb configurations. This is useful for if trying to play as say, an intelligent dog, as dogs typically have four legs and no arms. However the work is still ongoing, and not yet ready for a general release. Much of the foundational work can be seen in the currently named "Work in Progress Limb Stuff" mod which ships with the game.

#### Losing limb in combat: sure, but character gonna die (almost always)

> Not the first time I see people asking when the combat will be so dangerous you will be able to lose your beloved arm or leg. Bullet points:

Reviewer: ```suggestion
Often I see people asking when will combat be so dangerous that you will be able to lose your beloved arm or leg. Bullet points:
```

Main ways to lose a limb are:

> You having so badly damaged/infected a limb that your doctor/surgeon has no other way to save it but by cutting it - you have pretty high chances of survival, maybe even a guarantee if it's a faction doctor or you invest in your medicine really well;

Reviewer: ```suggestion
You have so badly damaged/infected a limb that your doctor/surgeon has no other way to save it but by cutting it - you have pretty high chances of survival, maybe even a guarantee if it's a faction doctor or you invest in the medicine skill heavily;
```

There are sure other ways, more technical, like mutating your arm into a crab claw or replacing it with a bionic drill, but that's another story.

In summary, the main things that stop us from introducing limb loss are, at first, a lack of NPC intelligence so they can help you if you experienced this loss, and second, a lot of our stuff is missing being tied to limb scores (something that you should have decreased because you have no limb). This is the main reason all limb stuff currently exists in a separate mod and is not part of the game.

> NB: This talks specifically about average human experience; mutants are less limited to this (tho a mouse or bear mutant would lose the limb in pretty similar fashion).

Author: ```suggestion
NB: This talks specifically about average human experience; mutants are less limited to this (though a mouse or bear mutant would lose the limb in pretty similar fashion).
```
